packages/stream_video/lib/open_api/video/coordinator/model/own_user_response.dart

```diff
@@ -272,14 +278,17 @@ class OwnUserResponse {
       // Note 2: this code is stripped in release mode!
       assert(() {
         requiredKeys.forEach((key) {
-          //assert(json.containsKey(key), 'Required key "OwnUserResponse[$key]" is missing from JSON.');
-          //assert(json[key] != null, 'Required key "OwnUserResponse[$key]" has a null value in JSON.');
+          assert(json.containsKey(key), 'Required key "OwnUserResponse[$key]" is missing from JSON.');
+          assert(json[key] != null, 'Required key "OwnUserResponse[$key]" has a null value in JSON.');
```

Comment:
Have you verified these keys now exist in the response?

Reply:
Tested those on dogfooding, works fine